test/fhir.test.js

```diff
@@ -1,5 +1,42 @@
-describe('fhir.js placeholder describe block', () => {
-  test('fhir.js placeholder test', () => {
-    expect(true).toBe(true);
+const { PatientSource } = require('../src/fhir');
+const testPatients = require('./fixtures/testPatients.json');
+const { testSetup, connection } = require('./setup');
+
+const TEST_PATIENT_IDS = testPatients.map(e => e.id);
+describe('Patient Source', () => {
+  beforeAll(async () => await testSetup(JSON.parse(JSON.stringify(testPatients))));
```

Comment:
Do we need to parse and stringify for a specific reason?

Reply:
Yeah, although there may be a more elegant workaround. Adding the resources to mongo populated the testPatient resources with the _id even in the stored const (not just in mongo), so the expects were failing because the mongo query strips off the _id before returning, but the testPatients consts still had them. So I just inserted a copy. Any thoughts?